MrE/src/main/java/frc/robot/OI.java

```diff
@@ -21,7 +21,7 @@ public class OI {
     private static SendableChooser<Command> autoChooser;
 
     private static SendableChooser<Integer> driverControlsChooser;
-    private static SendableChooser<Integer> controllerScalingChooser;
+    private static SendableChooser<DriveScaling> controllerScalingChooser;
```

Comment:
Whenever you see a class that has these brackets '<>' it means it's a typed class. It uses generics to allow developers to use any object they want.